forward_command_controller/include/forward_command_controller/multi_interface_forward_controller.hpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2021, PickNik, Inc.
+// Copyright (c) 2021, Stogl Robotics Consulting UG (haftungsbeschränkt) (template)
```

Comment:
just one please

Reply:
of course. This is the reason for draft PR :) Not everything is cleared, but rather auto-generated :)